metadata.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: spark-integration-hub-k8s
+display-name: Spark Integration Hub
```

Comment:
I believe we use K8s here, elsewhere, see [here](https://github.com/canonical/kafka-k8s-operator/blob/main/metadata.yaml#L4)